nix/cloud/actions.nix

```diff
@@ -0,0 +1,23 @@
+{
+  cell,
+  inputs,
+}: {
+  "hydra/ci" = {
+    task = "ci";
```

Comment:
how much effort would it be to migrate our current [ci workflow](https://github.com/input-output-hk/hydra/blob/master/.github/workflows/ci.yaml#L37) to this approach?

are there any pain points that you anticipate?

Reply:
Should be doable. Finding and updating the PR comment on GitHub is probably the trickiest part. We should provide some convenience for things like this in tullia. Let's merge this PR first and then gradually work on porting the other workflow steps.